train_wrapper.py

```diff
@@ -0,0 +1,34 @@
+"""Functionality for locally training files."""
+import torch
+import torch.nn as nn
+from torch.utils.data import DataLoader
+
+from app.ml_models.rnn.data_tools import WordLevelDataset
+from app.ml_models.rnn.vocabulary import Vocabulary
+from app.ml_models.rnn.train import train
+from app.ml_models.rnn.rnn_model import RNNAnna
+
+
+def run():
+    """Run a simple training loop."""
+    v = Vocabulary()
+    v.build(prefix=".", filename_source="data/familienamen.txt",filename_destination="vocabulary_extended.txt")
+    dataset = WordLevelDataset(prefix="data/", filename_datasets=["familienamen.txt"], vocabulary=v)
```

Comment:
Should we maybe make this a little bit more generic and give the filename_source and similarly the filename_datasets as arguments to the run function or even as argparse (default) arguments if we only run this manually from the command line?

Reply:
Yeees  I did put a TODO: argparse there, right?

Reply:
Fair, I skimmed over that too quickly.